src/components/AvatarGroup/AvatarGroup.driver.ts

```diff
@@ -0,0 +1,20 @@
+import {
+  BaseUniDriver,
+  baseUniDriverFactory,
+} from 'wix-ui-test-utils/base-driver';
+import { UniDriver } from 'wix-ui-test-utils/unidriver';
+import style from './AvatarGroup.st.css';
+
+export interface AvatarGroupDriver extends BaseUniDriver {
+  getAvatarCount(): Promise<number>;
+}
+
+export const avatarGroupDriverFactory = (
+  base: UniDriver,
+): AvatarGroupDriver => {
+  const getAvatarCount = () => base.$$(`.${style.root} > *`).count();
```

Comment:
another thing, I should have asked this before,
Let's use a `data-*` instead of `style.root`.
We sometimes have errors when using style imports inside drivers. 
It can make problems in e2e tests sometimes.
So maybe you can pass a `data-hook="avatar"` to the `Avatar`'s or something like that,
And then in this file you can use `base.$$('[data-hook="avatar"]').count()`?

Reply:
Did that;